lib/puppet/type/group.rb

```diff
@@ -108,7 +110,9 @@ def is_to_s(currentvalue)
     end
 
     newparam(:auth_membership, :boolean => true, :parent => Puppet::Parameter::Boolean) do
-      desc "whether the provider is authoritative for group membership."
+      desc "Whether the provider is authoritative for group membership. This
+        must be set to true to allow setting the group to empty e.g.
+        `members => [],`."
```

Comment:
It's actually very clear if you have the context of what Groups are in the sense of Windows. Setting a group to have no members then?